module/skill/Walk/data/config/Walk.yaml

```diff
@@ -2,25 +2,31 @@ log_level: INFO
 
 walk:
   # Time for one complete step (in seconds).
-  period: 0.35
+  period: 0.375
```

Comment:
tuning should go in another PR, but if you want to get it through in this at least update the PR title